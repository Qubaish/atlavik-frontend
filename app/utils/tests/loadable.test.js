import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Loadable from '../loadable';

describe('Loadable', () => {
    it('should render a Home page', () => {
        const container = shallow(
            <Loadable fallback={"...."}>
                <div>test</div>
            </Loadable>
        );
        expect(shallowToJson(container)).toMatchSnapshot();
    });
});