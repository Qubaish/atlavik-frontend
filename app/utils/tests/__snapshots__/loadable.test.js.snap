// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loadable should render a Home page 1`] = `<Component />`;
